vcd/resource_vcd_org_vdc_test.go

```diff
@@ -36,6 +36,19 @@ func TestAccVcdOrgVdcReservationPool(t *testing.T) {
 		// cause vDC ignores empty values and use default
 		"MemoryGuaranteed": "1",
 		"CpuGuaranteed":    "1",
+		// These parameters for Flex allocation.
```

Comment:
Suggestion for here and other occurrences in the file:
```suggestion
		// Below parameters are for Flex allocation model.
```

Reply:
changed